verification/example_config.toml

```diff
@@ -13,5 +13,10 @@ refresh_versions_schedule = "0 0 * * * * *"
 compilers_list_url = "https://solc-bin.ethereum.org/linux-amd64/list.json"
 
 [metrics]
+enabled = false
 endpoint = "/metrics"
 addr = "0.0.0.0:6060"
+
+[tracing]
+enable_jaeger = false
+jaeger_agent = "localhost:6831"
```

Comment:
I suggest to name the section `jaeger` and use `enabled` and `agent_addr` as keys.

As another option we may use `tracing.jaeger` as a subsection, but not sure whether it make sense now, as we do not have any other options for `tracing` section